<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutBrainSuiteDialog</class>
 <widget class="QDialog" name="AboutBrainSuiteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>460</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="visitBrainSuiteButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>224</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Visit the BrainSuite website</string>
   </property>
  </widget>
  <widget class="QLabel" name="brainSuiteCredit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>601</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string>see BrainSuiteSettings.cpp for text</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ClickLabel" name="newVersionAvailable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>741</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>Checking for latest version...</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="brainsuite.qrc">:/images/images/brainsuite_icon_128px.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="brainSuiteCreditLibs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>741</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>see BrainSuiteSettings.cpp for text</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickLabel</class>
   <extends>QLabel</extends>
   <header>clicklabel.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="brainsuite.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AboutBrainSuiteDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AboutBrainSuiteDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
